### ISSUE TITLE
Ajouter une REGEX de N° SAGE

### ISSUE THREAD

**Issue by celinebaulu** [Happykiller/vergo_puppet#2]
Exemple de N° SAGE : BTZ0001

Contient toujours 3 lettres MAJ + 4 chiffres

**Comment by celinebaulu** [comment]
{
    "regex":"\\b\\(?[A-Za-z]{2,3}\\d{4}\\)?\\b",
    "op": "REPLACE",
    "str": "[no_process][sage_id][/no_process]"
  }

**Comment by Happykiller** [comment]
`
{
    "regex":"\\b(?[A-Za-z]{2,3}\\d{4}\\)?\\b",
    "op": "REPLACE",
    "str": "[no_process][sage_id][/no_process]"
    }
`

**Comment by Happykiller** [comment]
{
    "regex":"\\b(?[A-Za-z]{2,3}\\d{4}\\)?\\b",
    "op": "REPLACE",
    "str": "[no_process][sage_id][/no_process]"
    }

**Comment by celinebaulu** [comment]
```
` {
    "regex":"\\b\\(?[A-Za-z]{2,3}\\d{4}\\)?\\b",
    "op": "REPLACE",
    "str": "[no_process][sage_id][/no_process]"
  }`
```

**Comment by Happykiller** [comment]
fix en 0.10.0
